pages/studentInstanceQuestionExam/studentInstanceQuestionExam.ejs

```diff
@@ -5,8 +5,8 @@
     <script type="text/x-mathjax-config">MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}});</script>
     <script type="text/javascript" src="/MathJax/MathJax.js?config=TeX-MML-AM_CHTML"></script>
     <script type="text/javascript" src="/javascripts/socket.io.js"></script>
-    <% if (question.type != 'Freeform') { %>
     <script type="text/javascript" src="/javascripts/lodash.min.js"></script>
+    <% if (question.type != 'Freeform') { %>
```

Comment:
Oh right, sorry, I mean lodash. Your PR shifts the lodash line above the `if Freeform` check. I think we both agree that the lodash line should stay, but inside the `if Freeform` block?